manifest.php

```diff
@@ -32,7 +32,7 @@
     'label' => 'QTI Portable Custom Interaction',
     'description' => '',
     'license' => 'GPL-2.0',
-    'version' => '6.0.0',
+    'version' => '6.1.0',
```

Comment:
Do we really need to up the minor version? Maybe just a patch?

Reply:
it was fixed